routes/seaice.py

```diff
@@ -0,0 +1,215 @@
+        lat: latitude for points or None for polygons
+        lon: longitude for points or None for polygons
+    Returns:
+        CSV response object
+    """
+
+    fieldnames = [
+        "year",
+        "month",
+        "concentration"
+    ]
+
+    months = [
+        "January",
+        "February",
+        "March",
+        "April",
+        "May",
+        "June",
+        "July",
+        "August",
+        "September",
+        "October",
+        "November",
+        "December",
+    ]
+
+    csv_pkg = list()
+    for key in data_pkg:
+        year = str(key[0:4])
+        month_index = int(key[5:]) - 1
+        month = months[month_index]
+        di = dict()
+        di["year"] = year
+        di["month"] = month
+        di["concentration"] = data_pkg[key]
+        csv_pkg.append(di)
+
+    metadata = "# Sea Ice Concentration is the percentage of sea ice coverage at the given latitude and longitude for each year and month.\n"
+    filename = "Sea Ice Concentration for " + lat + ", " + lon + ".csv"
+    return write_csv(csv_pkg, fieldnames, filename, metadata)
+
+
+def package_seaice_data(seaice_resp):
+    """Package the sea ice concentration data into a nested JSON-like dict.
+
+    Arguments:
+        seaice_resp -- the response(s) from the WCS GetCoverage request(s).
+
+    Returns:
+        di -- a nested dictionary of all SFE values
+    """
+    # initialize the output dict
+    di = dict()
+    for i in range(len(seaice_resp)):
+        di[f"{1850 + floor(i / 12)}-{str((i%12) + 1).zfill(2)}"] = seaice_resp[i]
+
+    return di
+
+
+def package_mmm_seaice_data(seaice_resp, start_year=None, end_year=None):
+    """Package the sea ice concentration data into a nested JSON-like dict of
+    min, mean, and max values.
+
+     Arguments:
+         seaice_resp -- the response(s) from the WCS GetCoverage request(s).
+         start_year -- starting year to find mmm
+         end_year -- ending year to find mmm
+
+     Returns:
+         di -- a nested dictionary of all sea ice concentration values
+    """
+    if None in seaice_resp:
+        return render_template("404/no_data.html"), 404
+
+    # initialize the output dict
+    di = dict()
+    start_index = 0
+    end_index = len(seaice_resp)
+    if start_year != None:
+        start_index = (int(start_year) - 1850) * 12
+    if end_year != None:
+        end_index = (int(end_year) - 1850) * 12
+
+    di["historical"] = dict()
+    di["historical"]["seaice_min"] = min(seaice_resp[start_index:end_index])
+    di["historical"]["seaice_max"] = max(seaice_resp[start_index:end_index])
+    di["historical"]["seaice_mean"] = round(np.mean(seaice_resp[start_index:end_index]))
+
+    return di
+
+@routes.route("/seaice/")
+@routes.route("/seaice/abstract/")
+def about_seaice():
+    return render_template("seaice/abstract.html")
+
+
+@routes.route("/seaice/point/")
+def about_seaice_point():
+    return render_template("seaice/point.html")
+
+
+@routes.route("/mmm/seaice/")
```

Comment:
I think I'd still recommend removing the mean/min/max endpoint for this variable.